### PR TITLE
slog duplicated to stdout && logger user addr for rpc call

### PR DESCRIPTION
**Pull request by sanlee42**

### CODE REVIEW

File: commons/logger/src/structured_log.rs [hunk 1]
```diff
@@ -73,11 +73,13 @@ fn create_default_root_logger(
...
 
     let decorator = slog_term::PlainDecorator::new(file);
-    let drain = slog_term::CompactFormat::new(decorator)
+    let file_drain = slog_term::CompactFormat::new(decorator)
         .use_custom_timestamp(timestamp_custom)
         .build()
         .fuse();
-
+    let decorator = slog_term::TermDecorator::new().build();
+    let io_drain = slog_term::CompactFormat::new(decorator).build().fuse();
```
Comment (jolestar) [inline commons/logger/src/structured_log.rs:81]: 确认下这个日志不会影响 console 输出

**Review by jolestar — APPROVED** [review]

File: commons/logger/src/structured_log.rs (hunk 1 above)
Comment (sanlee42) [reply commons/logger/src/structured_log.rs:81]: Screen Shot 2021-12-01 at 11 11 25 AM

seems to be working fine

Comment (jolestar) [reply commons/logger/src/structured_log.rs:81]: 是不是会同时输出到 console 里？

Comment (sanlee42) [reply commons/logger/src/structured_log.rs:81]: yes, block_relay_time is a slog info log

**Pull request merged**